ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6623,69 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+SEDPM:
```

Comment:
What is the use case for these on the unit you're looking at? l would not think of these as required points but perhaps they could be added as optional on existing abstract types.